x-pack/plugins/fleet/public/applications/fleet/components/fleet_server_instructions/steps/add_fleet_server_host.tsx

```diff
@@ -63,21 +66,42 @@ export const AddFleetServerHostStepContent = ({
     validate,
     inputs,
   } = fleetServerHostForm;
+  const theme = useTheme() as EuiTheme;
 
   const [isLoading, setIsLoading] = useState(false);
   const [submittedFleetServerHost, setSubmittedFleetServerHost] = useState<FleetServerHost>();
   const { notifications } = useStartServices();
   const { getHref } = useLink();
 
   const fleetServerHostsOptions = useMemo(
-    () =>
-      fleetServerHosts.map((fleetServerHost) => {
+    () => [
+      ...fleetServerHosts.map((fleetServerHost) => {
         return {
-          text: fleetServerHost.name,
+          inputDisplay: `${fleetServerHost.name} (${fleetServerHost.host_urls[0]})`,
           value: fleetServerHost.id,
         };
       }),
-    [fleetServerHosts]
+      {
+        icon: <EuiIcon type="plus" size="m" color="primary" />,
+        inputDisplay: (
+          <FormattedMessage
+            id="xpack.fleet.fleetServerSetup.addFleetServerHostBtn"
+            defaultMessage="Add new Fleet Server Hosts"
+          />
+        ),
+        dropdownDisplay: (
+          <EuiText size="relative" color={theme.eui.euiColorPrimary}>
+            <FormattedMessage
+              id="xpack.fleet.fleetServerSetup.addFleetServerHostBtn"
+              defaultMessage="Add new Fleet Server Hosts"
+            />
+          </EuiText>
+        ),
+        'data-test-subj': 'fleetServerSetup.addNewHostBtn',
+        value: '@@##ADD_FLEET_SERVER_HOST##@@',
```

Comment:
Is there a technical reason this gets a `value` formatted in this way?

Reply:
We need a value for the types to be correct, this one should not conflict with existing fleet server host id.